content/SCALE/SCALEUIReference/Storage/Pools/PoolCreateWizardScreens.md

```diff
@@ -0,0 +1,172 @@
+{{< /hint >}}
+
+If system disks have data exported from an pools on them, a warning displays with a checkbox for the pool name.
+
+{{< trueimage src="/images/SCALE/23.10/PoolCreationWizardGenInfoExportedPool.png" alt="General Info With Exported Pools" id="4: General Info with Exported Pools" >}}
+
+If system disks have non-unique serial numbers, a warning displays with additional fields.
+
+{{< trueimage src="/images/SCALE/23.10/PoolCreationWizardGenInfoNonUniqueSerialNums.png" alt="Non-Unique Serial Numbers Screen" id="5: Non-Unique Serial Numbers Screen" >}}
+
+**Allow non-unique serialed disks** has two radio buttons, **Allow** and **Don't Allow**.
+
+**Allow** permits using disks with non-unique serial numbers, such as those that can occur on virtual disks, and displays them as available disks on the **Data** wizard screen. 
+**Don't Allow** does not permit using disks with non-unique serial numbers.
+
+{{< /expand >}}
+
+### Data Wizard Screen
+The **Data** wizard screen provides the option to add disks to the data VDEV. You must add a data VDEV before adding other types of VDEVs to the pool.
+
+{{< trueimage src="/images/SCALE/23.10/PoolCreationWizardDataScreen.png" alt="Pool Creation Wizard Data Screen" id="6: Pool Creation Wizard Data Screen" >}}
+
+{{< expand "Click Here for More Information" "v" >}}
+**Layout** displays a dropdown list of VDEV layouts (**Stripe**, **Mirror**, **RAIDZ1**, **RAIDZ2**, and **RAIDZ3**). 
+
+{{< include file="/_includes/PoolCreationWizardCommonSettings.md" type="page" >}}
+{{< /expand >}}
+
+### Manual Selection Screen
+The **Manual Selection** screen allows you to add a VDEV of the type specified in the **Layout** field (on the **Data** and **Log** wizard screens), and to select the disks to add to the new VDEV. You can filter disks by type or size. 
+
+{{< expand "Click Here for More Information" "v" >}}
+**Add** displays VDEV area of the type specified in the **Layout** field. 
+Click on the system field to expand the dropdown list to show a list of available system disks. 
+Drag the number of disks to VDEV to add them. 
+
+**Save Selection** creates the VDEV and closes the window.
+
+{{< trueimage src="/images/SCALE/23.10/PoolCreationWizardManualSelectionScreen.png" alt="Manual Selection Screen" id="6: Manual Selection Screen" >}}
+
+#### Manual Selection Settings
+{{< truetable >}}
+| Setting | Description |
+|---------|-------------|
+| **Search** | Enter the disk name or other details to search for disks matching the specified value to filter available disks in the system. |
+| **Filer by Disk Type** | Filters the list of available disks displayed on the list of system disks by type (HDD or SSD). |
+| **Filter by Disk Size** | Filters the list of available disks displayed on the list of system disks by size. |
+| TrueNAS System | Click to expand and show the list of available disks in the system. Filter options change disks displayed on this list. |
```

Comment:
```suggestion
| **TrueNAS System** | Click to expand and show the list of available disks in the system. Filter options change disks displayed on this list. |
```

Reply:
TrueNAS System is not on the UI so it does not get bold emphasis.